packages/page-practice/lib/settings/KeyboardSettings.tsx

```diff
@@ -146,10 +146,24 @@ function KeyboardPreview(): ReactNode {
   );
 }
 
+/**
+ * Handle modifier keys only on keydown event for os and
+ * browser compatibility.
+ */
 function useDepressedKeys(): readonly string[] {
   const [depressedKeys, setDepressedKeys] = useState<readonly string[]>([]);
+  const modifierKeys = useRef(["CapsLock", "NumLock"]);
```

Comment:
The same comment.
When we create a `KeyEvent` instance from a `KeyboardEvent`, read the modifier state for these two keys and fill in the `KeyEvent.modifiers` array.